README.md

```diff
@@ -16,53 +16,62 @@ The Azure Functions Core Tools provide a local development experience for creati
 
 **v2** (master branch): Self-contained cross-platform package
 
-**v3**: Self-contained cross-platform package
+**v3**: (v3.x branch): Self-contained cross-platform package **(recommended)**
 
 ## Installing
 
 ### Windows
 
-To install runtime with npm:
+To download and install with MSI:
```

Comment:
I realize this takes the change beyond the initial scope you were planning, but can we make these headings? The document options are a bit difficult to read.

Reply:
Updated. Good suggestion